tools/iptb-plugins/filecoin/local/localfilecoin.go

```diff
@@ -374,7 +410,25 @@ func (l *Localfilecoin) Shell(ctx context.Context, ns []testbedi.Core) error {
 		nenvs = append(nenvs, fmt.Sprintf("NODE%d=%s", i, peerid))
 	}
 
-	return syscall.Exec(shell, []string{shell}, nenvs)
+	pid, err := l.getPID()
+	if err != nil {
+		return err
+	}
+
+	nenvs = filecoin.UpdateOrAppendEnv(nenvs, "FC_PID", fmt.Sprintf("%d", pid))
+	nenvs = filecoin.UpdateOrAppendEnv(nenvs, "FC_BINARY", l.binPath)
```

Comment:
These are to enable an easy way to use `dlv` when in side of a shell; eg `dlv attach $FC_PID $FC_BINARY`